DataPipeline.ipynb

```diff
@@ -0,0 +1,271 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>ABBV</td>\n",
+       "      <td>AbbVie</td>\n",
+       "      <td>Health Care</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>ACN</td>\n",
+       "      <td>Accenture</td>\n",
+       "      <td>Information Technology</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>498</th>\n",
+       "      <td>YUM</td>\n",
+       "      <td>Yum! Brands</td>\n",
+       "      <td>Consumer Discretionary</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>499</th>\n",
+       "      <td>ZBRA</td>\n",
+       "      <td>Zebra Technologies</td>\n",
+       "      <td>Information Technology</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>500</th>\n",
+       "      <td>ZBH</td>\n",
+       "      <td>Zimmer Biomet</td>\n",
+       "      <td>Health Care</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>501</th>\n",
+       "      <td>ZION</td>\n",
+       "      <td>Zions Bancorporation</td>\n",
+       "      <td>Financials</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>502</th>\n",
+       "      <td>ZTS</td>\n",
+       "      <td>Zoetis</td>\n",
+       "      <td>Health Care</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>503 rows × 3 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    Symbol          Company Name                Industry\n",
+       "0      MMM                    3M             Industrials\n",
+       "1      AOS           A. O. Smith             Industrials\n",
+       "2      ABT                Abbott             Health Care\n",
+       "3     ABBV                AbbVie             Health Care\n",
+       "4      ACN             Accenture  Information Technology\n",
+       "..     ...                   ...                     ...\n",
+       "498    YUM           Yum! Brands  Consumer Discretionary\n",
+       "499   ZBRA    Zebra Technologies  Information Technology\n",
+       "500    ZBH         Zimmer Biomet             Health Care\n",
+       "501   ZION  Zions Bancorporation              Financials\n",
+       "502    ZTS                Zoetis             Health Care\n",
+       "\n",
+       "[503 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "resp = requests.get('http://en.wikipedia.org/wiki/List_of_S%26P_500_companies')\n",
+    "soup = bs.BeautifulSoup(resp.text, 'lxml')\n",
+    "table = soup.find('table', {'class': 'wikitable sortable'})\n",
+    "\n",
+    "def SP500_scraper(table):\n",
```

Comment:
looks good to me, lets look into grouping these tickers by their GICS sectors in this week's sprint